doc/Gabc.tex

```diff
@@ -73,9 +73,13 @@ \subsection{Headers}
   \item There is a \verb=\greannotation= defined immediatly prior to \verb=\gregorioscore=.
   \item The \texttt{annotation} header field is defined.
   \end{itemize}
-\item[mode-modifier] The mode "modifier" of the piece. This may be any
+\item[mode-modifier] The mode ``modifier'' of the piece. This may be any
```

Comment:
I globally agree with this PR, I just think it would be useful to have a few convoluted mode examples and tell the user which part goes in the modifier and which in the differentia field... so that some conventions are respected and parsing is easier (for 3rd party softwares)